source/content/git-resolve-merge-conflicts.md

```diff
@@ -72,7 +72,7 @@ How you resolve a merge conflict depends on what type of conflict you're faced w
 
 ### Resolve delete/modify Conflicts
 
-To manually delete merge conflicts from the terminal, use the following commands in sequence. Start by identifying the file that is generating a delete error.
+When one commit removes a file and another commit modifies that file, a merge conflict is created.  To resolve such a delete/modify merge conflict from the terminal, use the following commands in sequence. Start by identifying the file that is generating a delete error.
```

Comment:
For copy edit: switch to active voice.

Reply:
@alexfornuto Thanks for the review!  Could you give an example of the kind of phrasing you mean?  I attempted to mimic other examples on this page, but I may have misunderstood.  (I do know passive vs. active voice in theory, I'm just not sure in this specific situation.)  Thank you!

Reply:
@alisonjo315 Sorry for the confusion, I added that while we were bug grooming as a note for ourselves. We'll handle the copy editing from here. Thanks!

Reply:
Ahhhh ok, gotcha, cool beans 😎

Reply:
Indeed, you did follow the existing phrasing on this page, which is also passive. I hope you don't mind if I push commits up to this fork/branch covering more than this section of the page.

Reply:
Oh of course not, by all means.  I appreciate the info, and I look forward to seeing how it turns out :)